src/slack_interaction_handler.ts

```diff
@@ -464,9 +464,10 @@ export async function handleCommandUnclaimed(
       arg = arg.substr(1); // strip off the # prefix
     }
     if (!(arg in slackChannelIds)) {
-      await SlackApiUtil.sendMessage(`Channel #${arg} not found`, {
-        channel: channelId,
-      });
+      await SlackApiUtil.sendEphemeralResponse(
```

Comment:
in addition to the fix, is this changing from a regular message to an ephemeral message? if so can you add this to the PR title and/or description?